components/filecoin/rs/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "filecoin-cxx"
+description = "FFI Interface to Filecoin"
+version = "0.0.1"
+edition = "2018"
+publish = false
+
+[dependencies]
+base64 = "0.12.3"
```

Comment:
The current stable release is `0.13.0` which we already use elsewhere in the tree. Would be nice to bump this to the same version.

Reply:
done, thanks